.github/workflows/e2e.yml

```diff
@@ -153,7 +156,6 @@ jobs:
             --param bundle_path=operators/test-e2e-operator/0.0.7-$SHORT_SHA \
             --param env=stage \
             --param submit=false \
-            --param pipeline_image=quay.io/redhat-isv/operator-pipelines-images:${{ github.sha }} \
```

Comment:
Why is this being removed? Wouldn't that mean the pipeline_image will use the default, which is the released image, and not what's on the PR?